charts/nri-prometheus/tests/configmap_test.yaml

```diff
@@ -19,3 +19,67 @@ tests:
             scrape_services: true
             transformations: []
             verbose: false
+
+  - it: creates the config map with lowDataMode.
```

Comment:
Now I understand when you quote Mateusz: "I only have to read the tests"

I was not able to understand the logic (and I know that it was me who merge this from the community)

Good test 👍